scex-core/src/main/scala/com/avsystem/scex/presentation/MathAttributes.scala

```diff
@@ -0,0 +1,120 @@
+package com.avsystem.scex.presentation
+
+import com.avsystem.scex.presentation.SymbolAttributes._
+import com.avsystem.scex.symboldsl.SymbolInfo
+
+object MathAttributes {
+  final val scalaMathAttributes: List[SymbolInfo[Attributes]] = attributes {
+    on { m: scala.math.type =>
+
+      m.IEEEremainder _ --> Attributes(documentation = "Returns the remainder resulting from the division of a specified number `x` by another specified number `y`.")
+
+      m.abs(_: Double) --> Attributes(documentation = "Returns absolute value of `x` argument.")
+      m.abs(_: Float) --> Attributes(documentation = "Returns absolute value of `x` argument.")
+      m.abs(_: Int) --> Attributes(documentation = "Returns absolute value of `x` argument.")
+      m.abs(_: Long) --> Attributes(documentation = "Returns absolute value of `x` argument.")
+
+      m.acos _ --> Attributes(documentation = "Returns the arc cosine of the `x` argument.")
+
+      m.addExact(_: Int, _: Int) --> Attributes(documentation = "Returns the sum of `x` and `y` arguments.")
+      m.addExact(_: Long, _: Long) --> Attributes(documentation = "Returns the sum of `x` and `y` arguments.")
+
+      m.asin _ --> Attributes(documentation = "Returns the arc sine of the `x` argument.")
+      m.atan _ --> Attributes(documentation = "Returns the arc tangent of the `x` argument.")
+      m.atan2 _ --> Attributes(documentation = "Returns the rectangular coordinates (x, y) converted to polar (r, theta).")
+
+      m.cbrt _ --> Attributes(documentation = "Returns the cube root of the `x` argument.")
+      m.ceil _ --> Attributes(documentation = "Returns the smallest double value that is greater than or equal to the `x` argument and is equal to a mathematical integer.")
+
+      m.copySign(_: Double, _: Double) --> Attributes(documentation = "Returns the first argument `magnitude` with the sign of the second argument `sign`.")
+      m.copySign(_: Float, _: Float) --> Attributes(documentation = "Returns the first argument `magnitude` with the sign of the second argument `sign`.")
+
+      m.cos _ --> Attributes(documentation = "Returns the cosine of the `x` argument.")
+      m.cosh _ --> Attributes(documentation = "Returns the hyperbolic cosine of the `x` argument.")
+
+      m.decrementExact(_: Int) --> Attributes(documentation = "Returns the `x` argument decremented by one.")
+      m.decrementExact(_: Long) --> Attributes(documentation = "Returns the `x` argument decremented by one.")
+
+      m.exp _ --> Attributes(documentation = "Returns the Euler's number raised to the power of the `x` argument.")
+      m.expm1 _ --> Attributes(documentation = "Return the Euler's number raised to the power of the `x` argument with one subtracted from it.")
```

Comment:
I'd rephrase this. The method calculates `exp(x) - 1`, but this description could also be understood as `exp(x-1)`